Analyzer/src/AnalyzeNjetsMinusOneCS.cc

```diff
@@ -0,0 +1,911 @@
+        bool pass_1l              = NGoodLeptons == 1;
+        bool pass_njet_pt45       = NGoodJets_pt45 >= 7;
+        bool pass_njet_pt45_1btag = NGoodBJets_pt45 >= 1;
+        bool pass_njet_pt45_2btag = NGoodBJets_pt45 >= 2;
+
+
+
+
+        // -------------------
+        // --- Fill Histos ---
+        // -------------------                        
+        const std::map<std::string, bool> cut_map_1l 
+        {
+            {"1l"                             , pass_1l                                                                  },
+            {"1l_ge6j"                        , pass_1l && pass_njet_pt45                                                },
+            {"1l_ge2b"                        , pass_1l && pass_njet_pt45_2btag                                          },
+            {"1l_ge6j_ge2b"                   , pass_1l && pass_njet_pt45 && pass_njet_pt45_2btag                        },
+            {"1l_ge6j_ge1b"                   , passBaseline1l_Good                                                      },                         
+            {"1l_ge6j_ge1b_ge5esm"            , passBaseline1l_Good && deepESM_val >= 0.5                                },                         
+            {"1l_ge6j_ge1b_ge5-6esm"          , passBaseline1l_Good && deepESM_val >= 0.5 && deepESM_val < 0.6           },                         
+            {"1l_ge6j_ge1b_ge6-7esm"          , passBaseline1l_Good && deepESM_val >= 0.6 && deepESM_val < 0.7           },                         
+            {"1l_ge6j_ge1b_ge7-8esm"          , passBaseline1l_Good && deepESM_val >= 0.7 && deepESM_val < 0.8           },                         
+            {"1l_ge6j_ge1b_ge8-95esm"         , passBaseline1l_Good && deepESM_val >= 0.8 && deepESM_val < 0.95          },                         
+            {"1l_ge6j_ge1b_ge8esm"            , passBaseline1l_Good && deepESM_val >= 0.8                                },                         
+            {"1l_ge6j_ge1b_ge95esm"           , passBaseline1l_Good && deepESM_val >= 0.95                               },                         
+            {"1l_ge6j_ge1b_l8esm"             , passBaseline1l_Good && deepESM_val <  0.8                                },                         
+            {"1l_ge6j_ge1b_d1"                , passBaseline1l_Good && deepESM_bin1                                      },                         
+            {"1l_ge6j_ge1b_d2"                , passBaseline1l_Good && deepESM_bin2                                      },                         
+            {"1l_ge6j_ge1b_d3"                , passBaseline1l_Good && deepESM_bin3                                      },                         
+            {"1l_ge6j_ge1b_d4"                , passBaseline1l_Good && deepESM_bin4                                      },                         
+        };
+
+        // Initialize Histograms
+        if(!initHistos)
```

Comment:
Since you have some "continues" above it is possible to have jobs that never initHistos. They will appear as failed jobs when you run condor.

Reply:
Yes, I know.  It's not intended to be functional for what I really want to do.  It currently is just for having a working example of the essential pieces running.